<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Smart Home Control Terminal</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>300</height>
       </size>
      </property>
      <widget class="QGroupBox" name="groupBoxCharts">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>341</width>
         <height>291</height>
        </rect>
       </property>
       <property name="title">
        <string>Charts</string>
       </property>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>130</y>
          <width>171</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>No chart selected</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>321</width>
          <height>261</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="widget_3" native="true">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>0</y>
         <width>471</width>
         <height>291</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="addDeviceButton">
        <property name="text">
         <string>Add Device</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="deviceDiscoveryToggle">
        <property name="text">
         <string>Enable Device Discovery</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="deviceTable">
      <column>
       <property name="text">
        <string>Image</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>IP Address</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Port</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Status</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Action</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
